src/olympia/accounts/views.py

```diff
@@ -162,7 +162,8 @@ def parse_next_path(state_parts):
         # but it only cares if there are too few so add 4 of them.
         encoded_path = state_parts[1] + '===='
         try:
-            next_path = base64.urlsafe_b64decode(str(encoded_path))
+            next_path = (
+                base64.urlsafe_b64decode(str(encoded_path)).decode('utf-8'))
```

Comment:
Wonder if calling `force_bytes` might be a bit more future-proof here to ensure we handle all kinds of weird encodings and always endup with something usable.